dist/js/nya-bs-select.js

```diff
@@ -553,7 +553,7 @@ nyaBsSelect.directive('nyaBsSelect', ['$parse', '$document', '$timeout', '$compi
         dropdownMenu.append(noSearchResult);
       }
 
-      if (tAttrs.actionsBox === 'true' && isMultiple) {
+      if (typeof tAttrs.actionsBox !== 'undefined' && isMultiple) {
```

Comment:
Same here. Actions Box will always show, unless you don't define the `actions-box` attribute in the HTML (or pass `undefined`, which is weird).